wagtail/snippets/tests/test_viewset.py

```diff
@@ -60,6 +62,36 @@ def test_no_model_set_or_passed(self):
         )
 
 
+class TestIncorrectConverter(SimpleTestCase):
+    def test_unknown_converter(self):
+        class BadViewSet(SnippetViewSet):
+            model = AdvertTag
+            url_converter = "foo"
+
+        with self.assertRaises(ImproperlyConfigured) as cm:
+            register_snippet(BadViewSet)
+        message = str(cm.exception)
+        self.assertEqual(
+            "BadViewSet is using an unknown URL converter",
+            message,
+        )
+
+
+class TestSnippetConverter(TestCase):
```

Comment:
Can probably merge this with `TestAdminURLs` a few lines below. We'll also need the same tests for `ModelViewSet`